<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>491</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>491</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>491</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить все</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>491</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить выделенное</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>491</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Название сорта</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Степень обжарки</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Молотый/в зернах</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Описание вкуса</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Цена(руб)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Объем упаковки(г)</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
